src/Platform.Handlers/src/Xamarin.Platform.Handlers/Core/IText.cs

```diff
@@ -2,7 +2,7 @@
 
 namespace Xamarin.Platform
 {
-	public interface IText : IFont
+	public interface IText : IView
```

Comment:
@hartez  and I were having a chat and this might not make sense when things with text might not be a view.

Reply:
EZ mentions that maybe this is because we named it wrong? `ITextView` might be better? 

>  "Does IText mean 'this is a control that displays text' or 'here are some properties of a thing that displays some text'?"

Reply:
Opened an issue on this: https://github.com/xamarin/Xamarin.Forms/issues/13832